src/model/keychain.js

```diff
@@ -28,7 +28,7 @@ export async function loadString(key, authenticationPrompt) {
       logger.log(`Keychain: loaded string for key: ${key}`);
       return credentials.password;
     }
-    logger.sentry(`Keychain: string does not exist for key: ${key}`);
+    captureMessage(`Keychain: string does not exist for key: ${key}`);
```

Comment:
This one is gonna drive us crazy because it triggers at least twice for every new users (addressKey and analyticsIdentifier doesn't exist the first time)